scripts/run_a_star.py

```diff
@@ -25,9 +25,11 @@ def main():
     graph = Graph(nodes_dict=nodes_dict)
     graph.draw_graph()
 
-    graph.a_star(start="M", end="HH")
-
-    graph.draw_result()
+    success = graph.a_star(start="M", end="HH")
+    if success:
+        graph.draw_result()
+    else:
```

Comment:
This would not be necessary then, but we can still have it. I'd however also raise an error there, that it is really clear that "run a star" did not return successfully.

Reply:
It's still good to have. Otherwise the return value of `a_star` wouldn't be used. And the algorithm can return false even it doesn't fail.

Reply:
I agree that the result should be considered, good catch! 
But since this targets printing, we could do 

```python
if not success:
    raise RuntimeError("A star algorithm did not find a path.")

graph.draw_result()
```

What do you think?

Reply:
Sure, sounds good.